pages/index.html

```diff
@@ -133,32 +133,37 @@ <h2 class="animate__animated animate__fadeInDown">Sequi ea ut et est quaerat</h2
           <div class="col-md-6 col-lg-3 d-flex align-items-stretch" data-aos="fade-up">
             <div class="icon-box icon-box-pink">
               <div class="icon"><i class="bx bxl-dribbble"></i></div>
-              <h4 class="title"><a href="">Lorem Ipsum</a></h4>
-              <p class="description">Voluptatum deleniti atque corrupti quos dolores et quas molestias excepturi sint occaecati cupiditate non provident</p>
+              <h4 class="title"><a href="">CyberStart America</a></h4>
+              <h6 class="subtitle">Participate in this amazing program!</p>
+              <p class="description">For beginners to seasoned professionals, Cyberstart America  provides participants with engaging challenges, a game, and a National Championship! This is an amazing opportunity, and to learn more, click on the link below.</p>
+              <div href="link"><a class="btn-get-started animate__animated animate__fadeInUp">CyberStart America Website</a></div>
             </div>
           </div>
 
           <div class="col-md-6 col-lg-3 d-flex align-items-stretch" data-aos="fade-up" data-aos-delay="100">
             <div class="icon-box icon-box-cyan">
               <div class="icon"><i class="bx bx-file"></i></div>
-              <h4 class="title"><a href="">Sed ut perspiciatis</a></h4>
-              <p class="description">Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur</p>
+              <h4 class="title"><a href="">Weekly Challenges</a></h4>
+              <h6 class="subtitle">Practice your secret agent skills!</p>
+              <p class="description">Come back and visit this website every Monday for a new challenge of the week! When you find the flag, submit it on our website. The first person to enter the correct flag will be featured on our Discord and website!</p>
```

Comment:
Another random tip: if you want to create a bigger space between titles and descriptions, you can use the "break" HTML tag:
```html
<br></br>
```